src/commands/functions/create.js

```diff
@@ -196,23 +195,76 @@ const pickTemplate = async function () {
 
 const DEFAULT_PRIORITY = 999
 
-/* get functions dir (and make it if necessary) */
-const ensureFunctionDirExists = function (context, flags, config) {
-  const functionsDir = config.build && config.build.functions
-  if (!functionsDir) {
-    context.log(`${NETLIFYDEVLOG} No functions folder specified in netlify.toml`)
-    process.exit(1)
+/**
+ * Get functions dir (and make it if necessary)
+ * @this FunctionsCreateCommand
+ * @returns {string | never} - functions directory or throws an error
+ */
+const ensureFunctionDirExists = async function () {
+  const { api, config, site } = this.netlify
+  const { log } = this
+  let functionsDirHolder = config.build && config.build.functions
+  let siteData = null
+  const siteId = site.id
+
+  // try to find functions_dir in site settings if not in user's netlify config
+  if (!functionsDirHolder) {
+    try {
+      siteData = await api.getSite({ siteId })
+    } catch (error) {
+      if (error.status === 404) {
+        error('Site not found')
+      } else {
+        error(error.message)
+      }
+    }
+
+    // https://open-api.netlify.com/#operation/getSite
+    functionsDirHolder = siteData?.build_settings?.functions_dir
   }
-  if (!fs.existsSync(functionsDir)) {
-    context.log(
+
+  if (!functionsDirHolder) {
+    log(`${NETLIFYDEVLOG} functions folder not specified in netlify.toml or app settings`)
+    const { functionsDir } = await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'functionsDir',
+        message: 'Enter the path where your functions should live:',
+        default: 'netlify/functions',
+      },
+    ])
+
+    try {
+      log(`${NETLIFYDEVLOG} updating app build settings with ${chalk.magenta.inverse(functionsDirHolder)}`)
```

Comment:
[pebble] I recommend changing `app build settings` to `site settings` to better reflect where the Functions directory setting is in the information architecture of the app UI.

```suggestion
      log(`${NETLIFYDEVLOG} updating site settings with ${chalk.magenta.inverse(functionsDirHolder)}`)
```